---a/net/minecraft/tileentity/IHopper.java
+++b/net/minecraft/tileentity/IHopper.java
@@ -3,8 +3,8 @@
 import net.minecraft.inventory.IInventory;
 import net.minecraft.world.World;
 
-public interface IHopper extends IInventory
-{
+public interface IHopper extends IInventory {
+
     World getWorld();
 
     double getXPos();
@@ -12,4 +12,4 @@
     double getYPos();
 
     double getZPos();
-}
+}
